modules/gitops-configuring-gitops-managed-resources-to-use-mounted-secrets.adoc

```diff
@@ -0,0 +1,106 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-gitops-managed-resources-to-use-mounted-secrets_{context}"]
+= Configuring {gitops-shortname} managed resources to use mounted secrets
+
+You must configure the {gitops-shortname} managed resources by adding volume mounts configuration to a deployment and configure the container pod to use the mounted secret. For example, consider that you want to add volume mounts configuration to the deployment of `app-taxi` application and the `100-deployment.yaml` file is in the `/environments/dev/apps/app-taxi/services/taxi/base/config/` directory.
+
+.Prerequisites
+
+* You have the AWS Secrets Manager resources stored in your {gitops-shortname} repository.
+* You have the Secrets Store Container Storage Interface (SSCSI) driver configured to mount secrets from AWS Secrets Manager.
+
+.Procedure
+
+. Add volume mounts configuration in the target resource:
```

Comment:
```suggestion
. Add volume mounts configuration in the target resource using secret provider class resources.
```

Reply:
@Srivaralakshmi You have two consecutive sentences ending in ":". That creates confusion - make the first sentence more complete and end it with a ".".

Reply:
@GroceryBoyJr wonderful suggestion! I like it!! Thank you!

Reply:
IBM STyle Guide also says: "Use a colon at the end of a sentence that introduces a subprocedure (equivalent to a nested ordered list) unless another sentence intervenes between the lead-in wording and the subprocedure. If a sentence intervenes, use a period to end both the lead-in sentence and the intervening sentence."

I have modified the sentences accordingly and resolves the issue. Thank you both for your suggestions.